cmd/libs/go2idl/client-gen/generators/generator-for-group.go

```diff
@@ -135,19 +144,20 @@ func set$.Group$Defaults(config *$.Config|raw$) error {
 	if err != nil {
 		return err
 	}
-	config.Prefix = "apis/"
```

Comment:
This is wrong, should be "/apis". See https://github.com/kubernetes/kubernetes/blob/master/pkg/client/unversioned/helper.go#L359